elytron/WFLY-19269-OCSP-stapling-support.adoc

```diff
@@ -0,0 +1,199 @@
+---
+categories:
+# Add any category for this proposal
+# if missing, add it to _data/widfly-categories and use its id
+---
+= [Experimental|Preview|Community|default]Template <INSERT TITLE HERE>
```

Comment:
I think we can go with Preview for now. The title here should be updated too.

Reply:
Fixed!